sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultVolumeSpec.java

```diff
@@ -3,32 +3,27 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.mesosphere.sdk.offer.Constants;
-import com.mesosphere.sdk.specification.validation.ValidationUtils;
+
+import java.util.regex.Pattern;
+
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 import org.apache.mesos.Protos;
 
-import javax.validation.constraints.NotNull;
-import javax.validation.constraints.Pattern;
-
 /**
  * This class provides a default implementation of the VolumeSpec interface.
  */
 public class DefaultVolumeSpec extends DefaultResourceSpec implements VolumeSpec {
 
-    private final Type type;
-
-    /** Regexp in @Pattern will detect blank string. No need to use @NotEmpty or @NotBlank. */
-    @NotNull
-    /*  No Slash character is allowed in container-path if using Persistent Volume
-         Mesos isolator/containerizer silently ignores mount operation:
-            mesos: src/slave/containerizer/mesos/isolators/filesystem/linux.cpp#L628
-            mesos:/src/slave/containerizer/docker.cpp#L473
-      Previous pattern:
-           @Pattern(regexp = "[a-zA-Z0-9]+([a-zA-Z0-9_-]*[/\\\\]*)*")
-                someDir/anotherDir in SandBox is not a valid path
+    /**
+     * Disallow slashes in the container path. If a slash is used, Mesos will silently ignore the mount operation for
+     * the persistent volume. See also:
+     *       mesos:src/slave/containerizer/mesos/isolators/filesystem/linux.cpp#L628
+     *       mesos:src/slave/containerizer/docker.cpp#L473
      */
-    @Pattern(regexp = "[a-zA-Z0-9]+([a-zA-Z0-9_-]*)*")
+    private static final Pattern VALID_CONTAINER_PATH_PATTERN = Pattern.compile("[a-zA-Z0-9]+([a-zA-Z0-9_-]*)*");
```

Comment:
Is a regexp really required here. Is a `contains` not sufficient? Alternatively, the comment should be updated to state that there are other characters that are also not allowed.

Reply:
Updated the comment